templates/cube/microcerts.html

```diff
@@ -0,0 +1,379 @@
+          </td>
+          {% elif module.status == "enrolled" %}
+          <td>
+            <div class="p-table--cube__contents u-align--right">
+              {% if has_prepare_material %}
+              <a class="p-button--neutral u-no-margin--right" href="{{ module.prepare_url }}">Prepare</a>
+              {% endif %}
+              <a style = "margin-left: 1rem;" class="p-button--positive" href="{{ module.take_url }}">Take</a>
+            </div>
+          </td>
+          {% elif module.status == "failed" %}
+          <td>
+            <div class="p-table--cube__contents u-align--right">
+              <a class="p-button--neutral u-no-margin--right" href="https://support.canonical.com">Request retry</a>
+            </div>
+          </td>          
+          {% elif module.status == "not-enrolled" %}
+          <td>
+            <div class="p-table--cube__contents u-align--right">
+              <a href="/cube/contact-us" class="p-button--positive cube-access js-invoke-modal">Apply for access</a>
+            </div>
+          </td>
+          {% elif module.status == "in-progress" %}
+          <td>
+            <div class="p-table--cube__contents u-align--right">
+              <a style = "margin-left: 1rem;" class="p-button--positive" href="{{ module.take_url }}">Resume</a>
+            </div>
+          </td>
+          {% endif %}
+        </tr>
+      {% endfor %}
+      </tbody>
+    </table>
+  </div>
+</section>
+{% endif %}
+
+{% if has_enrollments %}
+
+<section class="p-strip--square-suru has-cube is-slanted--top-right">
+  <div class="u-fixed-width">
+    <h2>Thank you for joining the beta</h2>
+    <p>Canonical are grateful for your participation. We look forward to hearing your feedback.</p>
+  </div>
+</section>
+
+{% else %}
+
+<section class="p-strip--square-suru has-cube is-slanted--top-right">
+  <div class="u-fixed-width">
+    <h2>Apply for beta access</h2>
+    <p>Canonical are seeking participants from the community for the CUBE beta test. If you would like to be considered for the beta, please enter your information in the form below. Canonical will reach out to you if you are selected.</p>
+    <a href="/cube/contact-us" class="p-button--positive cube-access js-invoke-modal">Apply for access</a>
+  </div>
+</section>
+
+{% endif %}
+
+<!-- Set default Marketo information for contact form below-->
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/cube-beta" data-form-id="3801" data-lp-id="7140" data-return-url="https://www.ubuntu.com/cube/thank-you" data-lp-url="https://pages.ubuntu.com/Contact_Us_Ubuntu_CUBEbetatest_Cube-contact-us-landing-page.html">
+</div>
+
+<script >
+const myObserver = new ResizeObserver((entries) => {
+  entries.forEach((entry) => {
+    const width = Math.floor(entry.contentRect.width);
+    if (width < 1035) {
+      entry.target.classList.remove("is-shallow");
+      document.getElementById("hero-content").style.marginTop = "2rem";
```

Comment:
The standard practise for targeting elements with JS is with a `js-` prefixed class, This separates styling from functionality and makes it easier to understand for the next developer.